src/sentry/models/eventuser.py

```diff
@@ -99,7 +102,7 @@ def find_similar_users(self, user):
             Project.objects.filter(
                 team__in=OrganizationMemberTeam.objects.filter(
                     organizationmember__user=user,
-                    organizationmember__organization=self.project.organization,
+                    organizationmember__organization__project=self.project_id,
```

Comment:
How does this relationship work? Because Organization isn't fk'd on Project. It's actually the opposite.

I feel like you'd need to do:

```python
organizationmember__organization=Project.objects.filter(id=self.project_id).values('organization_id')
```

Unless I'm missing some Django magic here.

Reply:
The SQL can still be expressed, I think Django does the right thing.

```
from sentry.models import Organization, Project
p = Project.objects.all()[0]
Organization.objects.filter(project__id=p.id)
[<Organization at 0x1099a3850: id=1L, owner_id=None, name=u'Sentry', slug=u'sentry'>]
```

Also "Querying in the opposite direction" here: https://docs.djangoproject.com/en/1.11/topics/db/examples/many_to_one/

Reply:
But I can change it if we want. My 5 year old memory is telling me this is fine though.